Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fedora
+MAINTAINER https://github.com/SatelliteQE
+
+RUN dnf -y install git python3-pip python3-devel && dnf clean all
+WORKDIR /root/manifester
+ENV MANIFESTER_DIRECTORY=/root/manifester/
+COPY . /root/manifester/
+RUN pip install .
+RUN cp manifester_settings.yaml.example manifester_settings.yaml
```

Comment:
Should this really be done in the dockerfile? the github workflow needs it, but I'd expect the runner to mount their config file.

Reply:
One of the big reasons for this is to make sure someone can run the dockerfile without mounts to see help output
```
$  docker run --rm manifester --help                                                                                        master 
Usage: manifester [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  get-manifest

```